translations/base-fr.yaml

```diff
@@ -460,30 +460,30 @@ buildings:
             description: Tourne une forme de 180 degrés.
     stacker:
         default:
-            name: Combineur
-            description: Combine deux formes. Si elles ne peuvent pas être combinées, la
-                forme de droite est placée sur la forme de gauche.
+            name: Assembleur
+            description: Assemblées deux formes. Si elles ne peuvent pas être assemblées,
+                la forme de droite est placée sur la forme de gauche.
     mixer:
         default:
             name: Mélangeur de couleur
             description: Mélange deux couleurs en utilisant la synthèse additive des
                 couleurs.
     painter:
         default:
-            name: Peintre
-            description: Colorie entièrement la forme venant de gauche avec la couleur
+            name: Station de peinture
+            description: Peint entièrement la forme venant de gauche avec la couleur
                 entrant en haut.
         mirrored:
-            name: Peintre
-            description: Colorie entièrement la forme venant de gauche avec la couleur
+            name: Station de peinture
+            description: Peint entièrement la forme venant de gauche avec la couleur
                 entrant en bas.
         double:
-            name: Peintre (double)
-            description: Colorie les deux formes venant de gauche avec la couleur entrant en
-                haut.
+            name: Station de peinture (double)
+            description: Peint entièrement les deux formes venant de gauche avec la
```

Comment:
same issue with the number here. The english string says « Colors the shapes on the left inputs... », so « Peint les formes venant de gauche... »

Reply:
I don't see the problem here